tests/compile/pxd_mangling_names.srctree

```diff
@@ -0,0 +1,43 @@
+# mode: compile
+# ticket: 2940
+
+PYTHON setup.py build_ext --inplace
+#PYTHON -c "import a; a.test()"
```

Comment:
Is there a reason not to run it? Importing it at least makes sure it's compiled and linked correctly.

Reply:
Oops, looks like a leftover '#' from my debugging session^^